netlify/functions/resources.ts

```diff
@@ -68,13 +68,45 @@ const resourcesHandler: Handler = async (event, context) => {
       body: tableToCsv(tableData),
     };
   } else if (resourceFilename === "findings.csv") {
-    const rawData = await readFile("./_data/findings/findings.csv", "utf8");
+    const findingsData = await getApiFindingsData();
+    let tableData: string[][] = [
+      [
+        "contest",
+        "handle",
+        "finding",
+        "risk",
+        "score",
+        "pie",
+        "split",
+        "slice",
+        "award",
+        "awardCoin",
+        "awardUSD",
+      ],
+      //contest,handle,finding,risk,score,pie,split,slice,award,awardCoin,awardUSD
+      ...findingsData.map((finding) => {
+        return [
+          "" + finding.contest,
+          finding.handle,
+          finding.finding,
+          finding.risk,
+          "" + finding.score,
+          "" + finding.pie,
+          "" + finding.split,
+          "" + finding.slice,
+          "" + finding.award,
+          finding.awardCoin,
+          "" + finding.awardUSD,
```

Comment:
`award` and `awardUSD` are getting truncated down to nearest integer when I test locally — unclear if that's in this code or something happening over on the API side.

<img width="1073" alt="Another numbers screenshot with CSV differences highlighted" src="https://user-images.githubusercontent.com/265902/232639213-599bda4a-058d-49b3-92fb-7fbd5b801ef2.png">

Reply:
Okay see https://github.com/code-423n4/api.code4rena.com/pull/534/files#r1169384518 for half of this fix and I'll make a new code suggestion for this side.

Reply:
To follow up, this was fixed with those api changes.